--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to create Makefile.in.
 
-SUBDIRS= pango modules pango-view examples docs tools tests
+SUBDIRS= pango modules pango-view examples tools tests
 
 EXTRA_DIST = 			\
 	autogen.sh		\
@@ -98,6 +98,4 @@
 pkgconfig_DATA += pangowin32.pc
 endif
 
-DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --enable-man --disable-doc-cross-references
-
 -include $(top_srcdir)/git.mk
--- a/configure.in
+++ b/configure.in
@@ -641,8 +641,6 @@
    fi
 fi
 
-GTK_DOC_CHECK([1.0])
-
 AC_ARG_ENABLE(man,
               AC_HELP_STRING([--enable-man],
                              [regenerate man pages from Docbook @<:@default=no@:>@]),
@@ -999,8 +997,6 @@
 modules/thai/Makefile
 modules/tibetan/Makefile
 examples/Makefile
-docs/Makefile
-docs/version.xml
 tools/Makefile
 tests/Makefile
 pango.pc
