diff --git a/pango/Makefile.am b/pango/Makefile.am
index fff14df..df311ce 100644
--- a/pango/Makefile.am
+++ b/pango/Makefile.am
@@ -132,7 +132,7 @@ pangoinclude_HEADERS =	   \
 pango-enum-types.h: s-enum-types-h
 	@true
 s-enum-types-h: @REBUILD@ $(pango_headers) Makefile
-	( cd $(srcdir) && glib-mkenums \
+	( cd $(srcdir) && ../debian/scripts/glib-mkenums \
 			--fhead "#ifndef __PANGO_ENUM_TYPES_H__\n#define __PANGO_ENUM_TYPES_H__\n\n#include <glib-object.h>\n\nG_BEGIN_DECLS\n" \
 			--fprod "/* enumerations from \"@filename@\" */\n" \
 			--vhead "GType @enum_name@_get_type (void);\n#define @ENUMPREFIX@_TYPE_@ENUMSHORT@ (@enum_name@_get_type())\n" 	\
@@ -145,7 +145,7 @@ s-enum-types-h: @REBUILD@ $(pango_headers) Makefile
 pango-enum-types.c: s-enum-types-c
 	@true
 s-enum-types-c: @REBUILD@ $(pango_headers) Makefile
-	( cd $(srcdir) && glib-mkenums \
+	( cd $(srcdir) && ../debian/scripts/glib-mkenums \
 			--fhead "#include <pango.h>" \
 			--fprod "\n/* enumerations from \"@filename@\" */" \
 			--vhead "GType\n@enum_name@_get_type (void)\n{\n  static GType etype = 0;\n  if (etype == 0) {\n    static const G@Type@Value values[] = {" 	\
