--- a/configure.in
+++ b/configure.in
@@ -127,12 +127,13 @@
 dnl Check for a working C++ compiler, but do not bail out, if none is found.
 dnl We use this for an automated test for C++ header correctness.
 dnl
-AC_CHECK_PROGS(CXX, [$CCC c++ g++ gcc CC cxx cc++ cl], gcc)
-AC_LANG_SAVE
-AC_LANG_CPLUSPLUS
-AC_TRY_COMPILE(,[class a { int b; } c;], ,CXX=)
-AM_CONDITIONAL(HAVE_CXX, test "$CXX" != "")
-AC_LANG_RESTORE
+dnl AC_CHECK_PROGS(CXX, [$CCC c++ g++ gcc CC cxx cc++ cl], gcc)
+dnl AC_LANG_SAVE
+dnl AC_LANG_CPLUSPLUS
+dnl AC_TRY_COMPILE(,[class a { int b; } c;], ,CXX=)
+dnl AM_CONDITIONAL(HAVE_CXX, test "$CXX" != "")
+dnl AC_LANG_RESTORE
+AM_CONDITIONAL(HAVE_CXX, false)
 
 #
 # Checks for HarfBuzz
